### ISSUE TITLE
Building mock on CentOS 10 runs into dependency isues

### ISSUE THREAD

**Issue by nkadel** [rpm-software-management/mock#1392]
### Short description of the problem
Building mock on CentOS 10 runs into dependency isues

### Output of `rpm -q mock`
Not applicable

### Any additional notes
Missing RPMs for compilation on CentOS 10 include:

* python3-backoff
* python3-pyroute2
* python3-rpmautospec-core
* python3-templated-dictionary

Those are all EPEL published RPMs. I could, in theory, spend some time adding them individually to my https://github.com/nkadel/mockrepo/ for building test builds of mock, but I figure you folks might get more traction than me getting the EPEL 10 repos available for use by "mock" on CentOS 10. And if our friends at EPEL are going to use mock to build RPMs, here is a starting list of missing requirements.

**Comment by nkadel** [comment]
I've published a working repo to build mock on CentOS Stream 10, at https://github.com/nkadel/mockrepo . I'm lying a bit to those configs, and generally building with AlmaLinux now, but with the AlmaLinux file replaced with the CentOS Stream 10 configs I've published at https://github.com/nkadel/nkadel-mock/ , it works well.

I've had difficulty publishing to EPEL before, the registration process got pretty confusing. Anyone is welcome to grab and review these for updates to EPEL. I do tend to roll back my RPM release numbers to "0.1" instead of the "1" used by EPEL, so I automatically get the EPEL versions when they're published.

**Comment by praiskup** [comment]
Thank you for opening the issue, but this doesn't seem to be fixable in Mock.  We should wait till the EPEL 10 appears, and then add a new set of configuration files.
https://discussion.fedoraproject.org/t/epel-10-proposal/44304

**Comment by nkadel** [comment]
Agreed. It's a heads up on the problem. I've gone ahead and added them to https://github.com/nkadel/mockrepo/ , which requires a bootstrap build to initially enable the initial copy of mock itself.
